src/main/resources/templates/email/programme-created/v1.0.0.html

```diff
@@ -0,0 +1,77 @@
+  >, with some vital pieces of information.</p>
+  <p><b>Placement Confirmation</b></p>
+  <p>At 12 weeks prior to your commencement in post, you will receive a placement notification from us. Please note this does not constitute an offer of employment but is a confirmation of the placement you will be due to undertake and mirrors the information that has been provided to your employing organisation.</p>
+  <p>At this point, your employing organisation will follow the NHS Employment Check Standards, details of which can be found <a href="https://www.nhsemployers.org/topics-0/employment-standards-and-regulation">here</a>.</p>
+  <p><b>National Training Number (NTN)</b></p>
+  <p>Subject to you completing all actions on TIS Self Service, on the first day <span
+      th:text="${not #strings.isEmpty(startDate)} ? ${startDate} : _"></span> of your programme, you will receive your National Training Number (NTN). It is also vitally important you connect to your Postgraduate Dean for Revalidation and ensure you are connected to the correct Deanery as your Designated Body on GMC Online. Please do <b>not</b> do this before <span
+      th:text="${not #strings.isEmpty(startDate)} ? ${startDate} : _"></span>. It is essential you do not connect to your trust or employing body. The notice containing your NTN will explain how to do this.</p>
+  <p><b>Pre-employment checks and processes</b></p>
+  <p>Both NHS England and your employing organisation understand that you will be busy in this transition period to the start of your upcoming training programme, but we wish to emphasise the importance of engaging with all organisations with the onboarding process.</p>
+  <p>Your employing organisation will be contacting you about pre-employment checks from 12 weeks prior to starting. It is essential you avoid delaying these processes so as not to cause any disruption to your employment or training programme. <b>The Medical Staffing teams will be processing a large number of new starters to their organisation and so any delay may impact their ability to add you to their employment systems, such as payroll.</b></p>
+  <p>Please ensure you contact your employing organisation if you have not heard anything <b>8 weeks</b> prior to your start date. Details about who this will be, will be contained in the placement confirmation email that you will receive 12 weeks before starting.</p>
+  <p>This timescale is in line with the <a href="https://www.hee.nhs.uk/our-work/medical-recruitment/code-practice-medical-recruitment">code of practice</a> agreed in conjunction with the BMA and NHS Employers.</p>
+  <p>Their pre-employment checks will include:</p><ul>
+  <li>Your right to work</li>
+  <li>If you have a criminal record</li>
+  <li>Employment history and references</li>
+  <li>Professional Registration and qualifications</li>
+  <li>Fitness to work</li>
+</ul>
+  <p><b><u>ACTIONs</u></b></p>
```

Comment:
I think this template could use a bit of a clean-up, but at the moment it is as defined in the ticket 🤷